include/vulkan/utility/vk_format_utils.h

```diff
@@ -227,7 +227,7 @@ inline bool vkuFormatIsPacked(VkFormat format);
 inline bool vkuFormatRequiresYcbcrConversion(VkFormat format);
 
 // Returns whether a VkFormat is XChromaSubsampled
-// This corresponds to formats with _422 in their name
+// This corresponds to formats with _422 or 420 in their name
```

Comment:
@spencer-lunarg does this match your original intent?

Reply:
yes,  "X Chroma Subsampled" is both, but `Y` is only 420